pkg/modules/modules_test.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2019 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !privileged_tests
```

Comment:
I was expecting this to be a privileged test. Can we really rely on modprobe without privileges?

Reply:
This file does not test the `FindOrLoadModules` function which calls modprobe. I will write a separate privilege test for that.

Reply:
Done.